src/main/react-front-end/src/views/Admin/ClientsInfoComponent.js

```diff
@@ -37,6 +44,41 @@ export default class ClientsInfoComponent extends Component{
 		});
 	}
 
+	// Shows the user a confirmation popup to confirm the update request.
+	enableOrDisableAdmin(event, row){
+		var popupMsg = ""
+		var isAdmin = event.target.checked;
+		if(isAdmin){
+			popupMsg = "Do you want to enable "+ row.firstName + " " + row.lastName+ " as admin.";
+			this.setState({showIsAdminPopup: true, adminChangePopupMsg:popupMsg, targetUser: row.email, isAdmin: true, firstName: row.firstName, lastName: row.lastName});
+		}
+		else{
+			popupMsg = "Do you want to disable "+ row.firstName + " " + row.lastName+ " as admin.";
+			this.setState({showIsAdminPopup: true, adminChangePopupMsg:popupMsg, targetUser: row.email, isAdmin: false, firstName: row.firstName, lastName: row.lastName});
+		}		
+	}
+
+	// The actual call to update the admin information is triggered after the user selects "Yes" in the confirmation popup
+	// The user information is retrieved from the state
+	enableOrDisableAdminUsingStateInfo(email, isAdmin){
+		enableOrDisableAdmin(email, isAdmin).then((resp)=>{
```

Comment:
enableOrDisableAdmin() method name and signature similar to local method created on line 48.
Please rename for better code readability.

Reply:
Incorporated. Renamed to enableOrDisableAdminApiCall